src/collections/_documentation/platforms/python/default-integrations.md

```diff
@@ -23,6 +23,22 @@ This integration registers with the interpreter's except hook system.  Through t
 any exception that is unhandled will be reported to Sentry automatically.  Exceptions
 raised interactive interpreter sessions will not be reported.
 
+### Options
+
+You can pass the following keyword arguments to `ExcepthookIntegration()`:
+
+* `always_run`:
+    
+  ```bash
+  $ python
+  >>> import sentry_sdk
+  >>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
+  >>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
+  >>> raise Exception("I will become an error")
+  ```
+
+  By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
```

Comment:
```suggestion
  By default, the SDK does not capture errors occurring in the REPL (`always_run=False`).
```

Reply:
ops, now fixing .. :) Bad typo

Reply:
No worries! This looks great. :) Thanks for all your work!